droid-hal-device.inc

```diff
@@ -425,6 +425,10 @@ sed -i -e '/^[[:space:]]*mount[[:space:]]/s/^/# Removed during droid-hal-device
 # remove the creation of /tmp to prevent wrong permissons 
 sed -i -e '/^[[:space:]]*mkdir[[:space:]]\/tmp[[:space:]]*/s/^/# Removed during droid-hal-device build : /' $RPM_BUILD_ROOT/*rc
 
+if [ $ANDROID_VERSION_MAJOR -ge "7" ]; then
+mkdir -p $RPM_BUILD_ROOT%{_libexecdir}/droid-hybris/system/etc
+cp -a %{android_root}/out/target/product/%{device}/system/etc/init $RPM_BUILD_ROOT%{_libexecdir}/droid-hybris/system/etc/.
```

Comment:
At least on my hybris-14.1 build only servicemanager.rc appears in that folder but of course we could change it so that only that file is copied because we haven't modified any other .rc files that might go there.